### ISSUE TITLE
Add Python 3.11 to tested versions

### ISSUE THREAD

**Issue by MattDMo** [danger89/fake-useragent#2]
Can you add Python 3.11 to the list of versions being tested? There shouldn't be any reason whatsoever that it shouldn't pass. The last beta [is out](https://pythoninsider.blogspot.com/2022/07/python-3110b5-is-now-available.html), and the Release Candidate process will begin soon.

**Comment by danger89** [comment]
I agree. In fact I need to fix all the test-cases also on older releases. Hellysmile used Travis in the past: https://travis-ci.org/github/hellysmile/fake-useragent.

However as you can see, even back in 2021 those tests were failing already. I might need to rewrite test-cases and debug existing test code. This will take a while to figure out all the legacy code.
